### PR TITLE
(Baobab) Add bisontry-messenger

### PR DESCRIPTION
**Pull request by jay-kim-kor**

### CODE REVIEW

File: bisontry-messenger/values.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+global:
+  name: bisontry-messenger
+  namespace: orakl
+
+image:
+  repository: asia-southeast1-docker.pkg.dev/bisontry/bisontry/bisontry-messenger
```
Comment (martinkersner) [inline bisontry-messenger/values.yaml:6]: Is the bisontry/bisontry correct?

**Review by martinkersner — APPROVED** [review]
LGTM!

**Review by nick-bisonai — APPROVED** [review]
lgtm!

File: bisontry-messenger/values.yaml (hunk 1 above)
Comment (jay-kim-kor) [reply bisontry-messenger/values.yaml:6]: Yes! first `bisontry` is GCP project name and second one is registry name. 😄

**Pull request merged**